.github/workflows/nightly.yml

```diff
@@ -94,3 +94,8 @@ jobs:
       with:
         report_paths: unittests.xml
 
+  deploy-alpha:
+    name: Deploy Nightly Alpha Build
+    uses: ./.github/workflows/alpha.yml
```

Comment:
Does this work? :) I think alpha.yml would need a `workflow_call` trigger set up in order to be called from another workflow. Currently it only has `workflow_dispatch` which is not quite the same (this one is used for running workflows manually).

Reply:
Ah okay, good to know! Will update that.

Related: wasn’t sure how to test nightly workflows from a branch...

Reply:
Fixed! (I think)